pom.xml

```diff
@@ -1521,7 +1521,7 @@
             <configuration>
               <rules>
                 <enforceBytecodeVersion>
-                  <maxJdkVersion>1.7</maxJdkVersion>
+                  <maxJdkVersion>1.8</maxJdkVersion>
```

Comment:
This change doesn't feel like something that should be part of a CassandraIO PR.